samples: cellular: modem_shell: Reconfigure location status LEDs for DKs (#15270)
nrfconnect/sdk-nrf: samples/cellular/modem_shell/README.rst

The LEDs have the following functions:

nRF91 series DKs:

>    * LED 2: Indicates the state of the GPIO pin when pulse counting has been enabled using the ``gpio_count enable`` command.
>    * LED 3: Indicates the LTE registration status.
>    * LED 4: Lit for five seconds when the current location has been successfully retrieved by using the ``location get`` command.

Reviewer: ```suggestion

* **LED 2** indicates the state of the GPIO pin when pulse counting has been enabled using the ``gpio_count enable`` command.
* **LED 3** indicates the LTE registration status.
* **LED 4** is lit for five seconds when the current location has been successfully retrieved by using the ``location get`` command.
```

> Thingy:91 series:

Reviewer: I don't think we have a decision from PMT about the naming. I guess we can just say Thingy:91 and Thingy:91 X.

The LEDs have the following functions:

nRF91 Series DKs:

   * LED 2: Indicates the state of the GPIO pin when pulse counting has been enabled using the ``gpio_count enable`` command.
   * LED 3: Indicates the LTE registration status.
   * LED 4: Lit for five seconds when the current location has been successfully retrieved by using the ``location get`` command.

> Thingy:91 Series RGB LED:
> 
>    1. LTE connected:
> 
>       * Default state constant BLUE
>       * Lit PURPLE for five seconds when the current location has been successfully retrieved by using the ``location get`` command.
> 
>    2. LTE disconnected:
> 
>       * Default state OFF
>       * Lit RED for five seconds when the current location has been successfully retrieved by using the ``location get`` command.

Reviewer: ```suggestion
Thingy:91 and Thingy:91 X RGB LED:

* LTE connected:

    * Default state constant blue.
    * Lit purple for five seconds when the current location has been successfully retrieved by using the ``location get`` command.

* LTE disconnected:

    * Default state OFF
    * Lit red for five seconds when the current location has been successfully retrieved by using the ``location get`` command.
```

The LEDs have the following functions:

nRF91 Series DKs:

>    * **LED 2** indicates the state of the GPIO pin when pulse counting has been enabled using the ``gpio_count enable`` command.
>    * **LED 3** indicates the LTE registration status.
>    * **LED 4** is lit for five seconds when the current location has been successfully retrieved by using the ``location get`` command.

Reviewer: ```suggestion
* **LED 2** indicates the state of the GPIO pin when pulse counting has been enabled using the ``gpio_count enable`` command.
* **LED 3** indicates the LTE registration status.
* **LED 4** is lit for five seconds when the current location has been successfully retrieved by using the ``location get`` command.
```

Thingy:91 and Thingy:91 X:

>    * **RGB LED:**

Reviewer 2: @peknis Do we need this RGB LED level here or could it be combined with the "Thingy:91 and Thingy:91 X:" line.

Reviewer: Well, that is what we had there at some point but it appeared again on its own line. I would be glad to get rid of an extra layer of bullets and have it on the same line with Thingies.

Reviewer 2: ok, so Matti can fix this. He is doing his first rebases so might be that something has gone wrong there.

Author: I considered that myself. But maybe it's better to align this with the DK chapter, as we may add more LED functionalities for Thingy in the future.

Author: Ok, so now "RGB LED" back on the same line with "Thingy:91 and Thingy:91 X"

      1. LTE connected:

         * Default state constant BLUE
         * Lit PURPLE for five seconds when the current location has been successfully retrieved by using the ``location get`` command.

      2. LTE disconnected:

         * Default state OFF

>          * Lit RED for five seconds when the current location has been successfully retrieved by using the ``location get`` command.

Reviewer: ```suggestion
Thingy:91 and Thingy:91 X:

* RGB LED:

  * LTE connected:

    * Default state constant blue.
    * Lit purple for five seconds when the current location has been successfully retrieved by using the ``location get`` command.

  * LTE disconnected:

    * Default state OFF.
    * Lit red for five seconds when the current location has been successfully retrieved by using the ``location get`` command.
```

>    * **RGB LED:**
> 
>       * LTE connected:
> 
>          * Default state constant blue.
>          * Lit purple for five seconds when the current location has been successfully retrieved by using the ``location get`` command.
> 
>       * LTE disconnected:
> 
>          * Default state OFF.
>          * Lit red for five seconds when the current location has been successfully retrieved by using the ``location get`` command.

Reviewer: ```suggestion
* **RGB LED:**

  * LTE connected:

    * Default state constant blue.
    * Lit purple for five seconds when the current location has been successfully retrieved by using the ``location get`` command.

  * LTE disconnected:

    * Default state OFF.
    * Lit red for five seconds when the current location has been successfully retrieved by using the ``location get`` command.
```
